common/shared.c

```diff
@@ -65,7 +65,7 @@ size_t buf_size, tx_size, rx_size;
 int rx_fd = -1, tx_fd = -1;
 char default_port[8] = "9228";
 
-char test_name[10] = "custom";
+char test_name[50] = "custom";
```

Comment:
Since we report the results separately, I added "fetch" and "base" to distinguish the atomic ops that are executed as both fetch atomic and base atomic ops.